src/main/java/mate/academy/springboot/datajpa/service/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+
+public interface ProductService {
+    Product save(Product product);
+
+    Product findById(Long id);
+
+    void deleteById(Long id);
+
+    Product update(Product product);
+
```

Comment:
```suggestion
    Product save(Product product);

    Product findById(Long id);

    void deleteById(Long id);
```
use save() for both create() and update()